lib/screens/explore_screens/flag/flag_screen.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/i18n/explore_screens/explore/explore.i18n.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_bloc.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_event.dart';
+import 'package:seeds/screens/explore_screens/flag/interactor/viewmodels/flag_state.dart';
+
+/// Explore SCREEN
```

Comment:
small detail.

Reply:
Probably we should start removing these coments on screens widgets, I did this in the past because of present the bloc  pattern in a undertable way.

Reply:
agreed, lets remove them